src/containers/facility/FacilityIntroduction.tsx

```diff
@@ -5,26 +5,52 @@ import { useParams } from 'react-router-dom';
 import { useAccommodationsAndOffers } from '../../hooks/useAccommodationsAndOffers.tsx';
 import { AccommodationWithId } from '../../hooks/useAccommodationsAndOffers.tsx/helpers';
 import { MediaItem } from '@windingtree/glider-types/types/win';
+import { Button } from '@mui/material';
 
 const Container = styled.div`
```

Comment:
We should use the `styled` from MUI, because eventually we will not support styled-components anymore.